### ISSUE TITLE
Non-text contrast ratio between keyboard focus and background of 'Edit' button is less than required 3:1 ratio.

### ISSUE THREAD

**Issue by msftedad** [angular/components#28807]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

Users with limited visions will not be able to see the focus between keyboard focus and background on 'Edit' clearly.

### Reproduction

Steps to reproduce:
1. Open URL: https://material.angular.io/components/button/overview
2. Expand sample selection and navigate to controls
3. Verify whether the Non-text contrast ratio between keyboard focus and background of 'Edit' button is less than required 3:1 ratio or not.
<img width="646" alt="218252_edit icon" src="https://github.com/angular/components/assets/93735775/9fcfe23e-a584-4bf9-8363-2fd2fc07bfb2">

### Expected Behavior

Non-text contrast ratio between focus and background of 'Edit' button should be greater than and equal to 3:1.

### Actual Behavior

Non-text contrast ratio between focus and background of 'Edit' button is 1.3:1 which is less than 3:1.

### Environment

- Angular:
- CDK/Material:
- Browser(s): Edge
Browser Version 122.0.2365.8
- Operating System (e.g. Windows, macOS, Ubuntu): 
Windows 11 22H2
OS Build: 23620.1000

**Comment by DBowen33** [comment]
![image](https://github.com/angular/components/assets/42016383/9f106a19-f2dc-4d83-9180-965ea6791f95)

Foreground color of three dots and background color of active/focused state pass color contrast ratio (7.42). 

To pass 1.4.11 Non-Text Contrast (AA), we only take into account the control graphic (three dots) against the immediate background (active/focused state). While the active/focused state circle technically does not meet color contrast against the white background, it is not included as one of the criterion to pass 1.4.11 because it is not part of the information indicating the button’s presence and purpose.

**Comment by msftedad** [comment]
@DBowen33, Thanks for the information.As per the discussion with PO team we have closed our issue. You can close this issue.

**Comment by essjay05** [comment]
Confirming that this issue is obsolete as of 12/19/2024 based on current theme/colors from [Angular Components Button Demo](https://material.angular.io/components/button/overview).

Screenshot for comparison below shows that the foreground HEX #44474e : background HEX #e8e9e9 (or #44474e at .12 opacity) has a contrast ratio of 7.65.
![Image](https://github.com/user-attachments/assets/cbda7bcb-77b7-4625-b1bb-9abcd680e46a)
